x-pack/plugins/security_solution/public/common/components/navigation/tab_navigation/index.tsx

```diff
@@ -45,6 +47,7 @@ const TabNavigationItemComponent = ({
       isSelected={isSelected}
       href={appHref}
       onClick={handleClick}
+      append={isBeta && <EuiBetaBadge label={BETA} size="s" />}
```

Comment:
Could you please add a unit test where make sure that the beta badge was added only if isBeta property is true?